python/source/org/libreoffice/ide/eclipse/python/OOoPythonPlugin.java

```diff
@@ -88,6 +92,9 @@ public void start(BundleContext pContext) throws Exception {
     public void stop(BundleContext pContext) throws Exception {
         super.stop(pContext);
         sPlugin = null;
+        if (PydevRemoteDebuggerServer.isRunning()) {
+            PydevRemoteDebuggerServer.stopServer();
+        }
```

Comment:
This is done so that the PyDev Debug server is closed, once the eclipse is closed.